.github/workflows/wikiman.yml

```diff
@@ -0,0 +1,21 @@
+name: wikiman
```

Comment:
I think this can be a string with spaces etc, so something more descriptive could be cool maybe? What do you think?

Reply:
Changed this to "wikiman daily run"